nuit/utils.py

```diff
@@ -13,10 +13,24 @@ def _closure(fn):
         return fn.__closure__
 
 
+def _code(fn):
+    '''
+    Python 2 and Python 3 compatibility function
+    '''
+    try:
+        return fn.func_code
+    except AttributeError:
+        return fn.__code__
+
+
 def has_closure(fn):
     return hasattr(fn, 'func_closure') or hasattr(fn, '__closure__')
 
 
+def has_code(fn):
```

Comment:
Not sure, can't find many references to how to tell if the attribute is supposed to be there or not. It's not listed on the inspect page.